.github/workflows/trunk-upgrade-and-check.yml

```
@@ -0,0 +1,24 @@
+name: Daily Trunk Upgrade and Check
+
+on:
```

YAMLlint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)